kurtosis-module/impl/grafana/grafana_launcher.go

```diff
@@ -49,11 +39,11 @@ type dashboardProvidersConfigTemplateData struct {
 
 func LaunchGrafana(
 	enclaveCtx *enclaves.EnclaveContext,
-	datasourceConfigTemplate *template.Template,
-	dashboardProvidersConfigTemplate *template.Template,
+	datasourceConfigTemplate string,
+	dashboardProvidersConfigTemplate string,
 	prometheusPrivateUrl string,
 ) error {
-	artifactUuid, err := getGrafanaConfigDirArtifactUuid(
+	artifactUuid, uploadArtifactUuid, err := getGrafanaConfigDirArtifactUuid(
```

Comment:
what's the difference between `artifactUuid` and `uploadArtifactUuid`? seems like we want better naming for these

Reply:
good catch, this was just me being lazy while writing and reviewing my own code. fixed it.